### DISCUSSION TITLE
use alternative frontend frameworks

### DISCUSSION THREAD

**Discussion by cemkaan** [refinedev/refine#6045] [Ideas]
### Is your feature request related to a problem? Please describe.

I prefer to use Vue or Svelte frontend 
Is it possible to adapt , Vue or Svelte in refine system?
[Adonis ](https://docs.adonisjs.com/guides/getting-started/installation#inertia-starter-kit)uses [inertia](https://inertiajs.com/) to accomplish this feature.

### Describe alternatives you've considered

Is it possible to adapt , Vue or Svelte in refine system?
[Adonis ](https://docs.adonisjs.com/guides/getting-started/installation#inertia-starter-kit)uses [inertia](https://inertiajs.com/) to accomplish this feature.

### Additional context

.vue .svelte .astro is cleaner than tsx

### Describe the thing to improve

Adding additional front-end layer will make refine better.

https://docs.adonisjs.com/guides/views-and-templates/introduction

**Comment by BatuhanW** [comment]
Hey @cemkaan thanks for the discussion! Refine core is built on top of react-hooks, so I assume it would be hard to share core package for other frameworks. Could you elaborate more on `[Adonis ](https://docs.adonisjs.com/guides/getting-started/installation#inertia-starter-kit)uses [inertia](https://inertiajs.com/) to accomplish this feature.` ?

**Reply by cemkaan** [reply]
Currently I am interested in building with refine , soon I hope I will get into usage of Adonis in refine.
